projects/swimlane/ngx-charts/src/lib/common/axes/x-axis-ticks.component.ts

```diff
@@ -123,7 +162,55 @@ export class XAxisTicksComponent implements OnChanges, AfterViewInit {
 
   update(): void {
     const scale = this.scale;
+    this.adjustedScale = this.scale.bandwidth
+      ? function (d) {
+          return this.scale(d) + this.scale.bandwidth() * 0.5;
+        }
+      : this.scale;
     this.ticks = this.getTicks();
+    const sign = this.orient === Orientation.Top || this.orient === Orientation.Right ? -1 : 1;
+    this.tickSpacing = Math.max(this.innerTickSize, 0) + this.tickPadding;
+
+    switch (this.orient) {
+      case Orientation.Bottom:
+        this.transform = function (tick) {
+          return 'translate(' + this.adjustedScale(tick) + ',0)';
+        };
+        this.textAnchor = TextAnchor.Middle;
+        this.x2 = this.innerTickSize * sign;
+        this.x1 = this.tickSpacing * sign;
+        this.dx = sign < 0 ? '0em' : '.71em';
+        break;
+      case Orientation.Left:
+        this.transform = function (tick) {
+          return 'translate(0,' + this.adjustedScale(tick) + ')';
+        };
+        this.textAnchor = TextAnchor.End;
+        console.log('ANCHOR' + this.textAnchor);
```

Comment:
Let's remove the console.log please.

Reply:
fixed in 7c46f55061e0bb08afaf843f724c6c5348a99071